metadrive/envs/base_env.py

```diff
@@ -586,7 +586,8 @@ def export_scenarios(
         scenario_index: Union[list, int],
         time_interval=0.1,
         verbose=False,
-        render_topdown=False
+        render_topdown=False,
+        return_done_info=False
```

Comment:
Why default to False? I think we always ask for returning done info in old code.

Reply:
It will bring some compatibility problems with the test. It is fine. I will solve this and set the default value to True